dll/directx/wine/wined3d/device.c

```diff
@@ -1181,7 +1184,12 @@ HRESULT CDECL wined3d_device_init_gdi(struct wined3d_device *device,
     return WINED3D_OK;
 
 err_out:
+#ifndef __REACTOS__
     wined3d_swapchain_decref(swapchain);
+#else
+    if (swapchain)
+        wined3d_swapchain_decref(swapchain);
```

Comment:
You shouldn't need to duplicate this to fix the warning. Just have both lines at the same indent level, i.e.
```
#ifdef __REACTOS__
    if (swapchain)
#endif
    wined3d_swapchain_decref(swapchain);
```

Reply:
Updated.